src/examples/Layout/nestedLayout.tsx

```diff
@@ -89,7 +89,7 @@ export function MyLayout({ Component, pageProps }: any) {
           <a href="/asdfasdfasdfasdf">A Route that does not exist</a>
         </li>
       </ul>
-      <Component {...pageProps} />
+      {router_outlet}
     </MasterLayout>
   );
```

Comment:
This is the new way to use the router... the router passes a router_outlet prop (which is a signal) the can be placed in you template/layout component (and soon to be leveraged with nested routing)